app/models/user.rb

```diff
@@ -14,7 +14,11 @@ class User
   # :timeoutable
 
   # Relationships
+<<<<<<< HEAD
```

Comment:
Let's keep the `has_and_belongs_to_many` version of this -- ditch lines 17-19 and 21.